lib/field_struct/avro_schema/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module FieldStruct
   module AvroSchema
-    VERSION = '0.1.22'
+    VERSION = '0.1.3'
```

Comment:
Version 0.1.3 < 0.1.22. So you'd either go to 0.1.23 or 0.2.0, but if you're doing semver then 0.1.23 seems best.

Reply:
Breakage and compatibility are two different things. If you're expecting it to be API-compatible (you are) then semver wants you to bump the patch version. But because 0.1.3 is less than 0.1.22, you're actually assigning an older version that will confuse package managers.

Reply:
I do think these changes warrant a 0.2.0 version

Reply:
Bumped to version 0.2.0 :)